src/App.tsx

```diff
@@ -64,7 +70,19 @@ function App() {
                     stepForwards={actions.stepForwards}
                     stepTo={actions.chooseStep}
                     stepMeForwards={actions.stepMeForwards}
+                    disabled={isRecording}
                   />
+                  {process.env.NODE_ENV === "development" && (
```

Comment:
If you add this as a separate Export tab, then this should be usable on the netlify preview link - and we can get people to kick the tyres and give us some feedback on how the flow works

Reply:
Ah, sure, I can do that